en/config/actuators.md

```diff
@@ -1,96 +1,250 @@
+
+![Geometry motor: standard vtol](../../assets/config/actuators/qgc_geometry_standard_vtol_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate the direction in which the motor moves the vehicle (for a standard VTOL, the hover motors are usually set "upwards" and the pusher motor is set to "forwards").
+
+- `Axis`: One of `Upwards`, `Downwards`, `Forwards`, `Backwards`, `Leftwards`, `Rightwards`, `Custom`
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
+
+#### Motor Geometry: Other Vehicles
+
+Other vehicle types will define an appropriate motor geometry for their frame type.
+Once again these motors will generally have the same kinds of properties as shown above.
+
+For example, a fixed-wing vehicle may just have a single pusher moter, while a rover with differential steering will have a motor for throttle and for steering.
+
+#### Bidirectional Motors
+
+Some vehicles may use bidirectional motors (i.e. motor supports spinning in both directions)
 For example, ground vehicles that want to move forwards and backwards, or VTOL vehicles that have pusher motors that go in either direction.
 
 If bidiectional motors are used, make sure to select the **Reversible** checkbox for those motors (the checkbox is displayed as an "advanced" option).
 
-![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible_marked.png)
 
 Note that you will need to also ensure that the ESC associated with bidirectional motors is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
 
 
+### Control Surfaces Geometry
+
+### Control Surface Direction (Conceptual)
+
+![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
+
+> Example Control Surface Geometry setup for a Tiltrotor VTOL
+
+:::note
+It is extremely important that we decouple the concept of control surface, servo and motor tilt directions in control allocation from the actual servo output / connections in the physical vehicle.
+
+That means that the Control Allocation Control Surface & Servo settings are physical vehicle independent. Therefore when we think about the surfaces moving, we imagine the ideal vehicle as shown below that moves actuators in the given direction!
+:::
+
+![Control Surface Deflections](../../assets/config/actuators/plane_control_surface_convention.png)
+
+Control surfaces use the following deflection direction convention:
+
+#### Horizontal Control Surface (e.g. Aileron)
+
+> Upwards movement equals Positive deflection
+
+'Pitch UP' control input should command positive deflection to both ailerons
```

Comment:
All of this stuff is very interesting and has nothing to do with configuration of these surfaces. What we need here is an outline of how you set the scaling values. When are they 1? When are they -1. What does 0 mean. Why 0.5?